### ISSUE TITLE
No-Vary-Search: WHATWG list notation « » is unfamiliar to some readers

### ISSUE THREAD

**Issue by jeremyroman** [httpwg/http-extensions#2907]
Migrated from https://github.com/jeremyroman/http-no-vary-search/issues/4

**Comment by domenic** [comment]
I noticed it used in https://johannhof.github.io/draft-annevk-johannhof-httpbis-cookies/draft-annevk-johannhof-httpbis-cookies.html

**Comment by domenic** [comment]
I am unsure what to do with this.

The notation appears several times in the document, always with single-element lists in examples:

```
no-vary params: « "`a`" »
vary params: « "`x`" »
will result in a URL search variance whose vary params are « "`é 気`" »
```

We could be verbose and replace this with `` a list with the single element "`a`" ``, etc.

We could try to clarify with something like ``the list « "`a`" » ``.

We could just throw up a "This document depends on Infra" line, like https://johannhof.github.io/draft-annevk-johannhof-httpbis-cookies/draft-annevk-johannhof-httpbis-cookies.html#name-conventions does. That is probably a good idea anyway, since we're taking, e.g., our definition of strings from there. Or maybe even "This document depends on Infra, including the depiction of list literals using « » syntax."

One could imagine using different punctuation, like `(` and `)`, but that seems gratuitously divergent with the existing spec ecosystem to me. If there's precedent to point to in IETF specs though, then maybe it's fine.

I welcome any advice from the IETF community, including @martinthomson who originally filed this over in https://github.com/jeremyroman/http-no-vary-search/issues/4.

**Comment by jeremyroman** [comment]
It does have a "This document depends on Infra" line:

> This document also adopts some conventions and notation typical in WHATWG and W3C usage, especially as it relates to algorithms. See [[WHATWG-INFRA](https://httpwg.org/http-extensions/reschke/resumable1/draft-ietf-httpbis-no-vary-search.html#WHATWG-INFRA)].

**Comment by martinthomson** [comment]
Right.  Though, as I said, infra is massive and pointers to specific items would help (you don't need much from that spec).
